enhancements/sig-architecture/98-cluster-permission/README.md

```diff
@@ -0,0 +1,226 @@
+`ManifestWork` API will ensuring, updating, removing the RBAC resources
+from each of the managed cluster. 
+The `ClusterPermission` API will protect these distributed RBAC resources from unexpected 
+modification and removal.
+
+### Enhance the usability of the existing ManagedServiceAccount API
+
+After ensuring the service-accounts' presence on each managed cluster using the existing
+[ManagedServiceAccount](https://github.com/open-cluster-management-io/managed-serviceaccount)
+API, the `ClusterPermission` can reference the `ManagedServiceAccount` and use that
+as the Subject for the RBAC resources. This will help regulate the access level for
+the `ManagedServiceAccount`.
+
+### New API instead of just using ManifestWork API
+
+There are several reasons why we should create a new API instead of using `ManifestWork`:
+- `ManifestWork` API is meant to be a multicluster workload delivery primitive type.
+- `ClusterPermission` API is a higher level concept that users can understand its multicluster
+RBAC resources scope easily.
+- `ManifestWork` API payload does not have RBAC specific fields definitions.
+- `ClusterPermission` can determine the `ManagedServiceAccount` subject and create `ManifestWork`
+with the correct RoleBinding/ClusterRoleBinding subject.
+
+## Goals & Non-Goals
+
+### Goals
+
+- Manage the lifecycle of RBAC resources across multiple cluster.
+- Allows the typical Subject Kind: Group, ServiceAccount, and User
+for RoleBinding and ClusterRoleBinding.
+- Allows the `ManagedServiceAccount` as Subject Kind 
+for RoleBinding and ClusterRoleBinding.
+
+### Future goals
+
+- It is currently assumed that the user of `ClusterPermission` is either a cluster admin or
+a user who can create `ClusterPermission` in the hub cluster's managed "cluster namespace".
+It will be useful if there is an approval process that a cluster admin can approve,
+which will allow for other users to create `ClusterPermission` resources.
+- Support creating a RoleBinding/ClusterRoleBinding without defining any Role/ClusterRole
+rules.
+
+## Design
+
+### Component & API
+
+We purpose to adding a new custom resource named
+`ClusterPermission` introduced into OCM by this proposal:
+
+A sample of the `ClusterPermission` will be:
+
+```yaml
+apiVersion: rbac.open-cluster-management.io/v1alpha1
+kind: ClusterPermission
+metadata:
+  name: my-managed-rbac
+  namespace: cluster1
+spec:
+  clusterRole:
+    rules:
+    - apiGroups: ["apps"]
+      resources: ["deployments"]
+      verbs: ["update"]
+  roles:
+  - namespace: default
+    rules:
+    - apiGroups: ["apps"]
+      resources: ["deployments"]
+      verbs: ["update"]
+  - namespaceSelector:
+      matchExpressions:
+      - key: foo.com/managed-state
+        operator: In
+        values:
+        - managed
+    rules:
+    - apiGroups: [""]
+      resources: ["configmaps"]
+      verbs: ["update"]
+  subject:
+    apiGroup: authentication.open-cluster-management.io
```

Comment:
will this create a clusterrolebinding for clusterrole? What if I want to create a rolebinding for clusterrole?

Reply:
This is a great point. Added example of how to handle rolebinding and clusterrolebinding separately.